File: python/bee1074.py
n = int(input())
cont = 0
while cont < n:
    num = int(input())
    if num == 0:
        print('NULL', end='')
    else:
        if num % 2 == 0:
            print('EVEN', end=' ')
        else:
            print('ODD', end=' ')
        if num > 0:
            print('POSITIVE', end='')
        else:
            print('NEGATIVE', end='')
    print()
    cont += 1

File: javascript/bee1074.js
const inputs = require("fs").readFileSync("stdin", "utf8").split("\n").map(Number);
const [n, ...numbers] = inputs;

numbers.forEach((number) => {
	if (number === 0) {
		console.log("NULL");
	} else {
		const parity = number % 2 === 0 ? "EVEN" : "ODD";
		const sign = number > 0 ? "POSITIVE" : "NEGATIVE";
		console.log(`${parity} ${sign}`);
	}
});
